apps/src/templates/census2017/YourSchool.jsx

```diff
@@ -85,18 +98,31 @@ class YourSchool extends Component {
         {!this.props.hideMap && (
            <div>
              <h1 style={styles.heading}>
-               Put your school on the map
+               Does your school teach Computer Science?
              </h1>
              <h3 style={styles.description}>
-               {i18n.yourSchoolMapDesc()}
-               If you are located in the US, please <a href="#form">fill out the form below</a>.
-               If you are outside the US, <a href="/learn/local">add your school here</a>.
+               Find your school on the map to see if computer science is already being offered.
```

Comment:
Maybe I missed this, but I thought every school was on the map, but color coded based on whether we have info or not. I would expect this messaging to be more like "Find your school on the map to see if we have information about whether computer science is offered. If your school is <whatever color means we don't know> fill out the survey to to help us learn more".

Reply:
Every school that is in the `schools` table that has location information is on the map. There are schools that aren't in our dataset and there are times when people cannot find their school even if it is there.